src/shared_modules/router/src/remoteStateHelper.hpp

```diff
@@ -45,6 +45,7 @@ class RemoteStateHelper final
         socketClient->connect(
             [&](const char* body, uint32_t bodySize, const char*, uint32_t)
             {
+                // LCOV_EXCL_START
```

Comment:
Question: Why is not possible to test this code?

Reply:
I've just added the following tests for this class:
```
[----------] 4 tests from RemoteStateHelperTest
[ RUN      ] RemoteStateHelperTest.TestSendInvalidMessageType
RemoteProvider: Invalid result: Invalid message type
[       OK ] RemoteStateHelperTest.TestSendInvalidMessageType (3 ms)
[ RUN      ] RemoteStateHelperTest.TestSendEmptyMessage
RemoteProvider: Invalid result: [json.exception.out_of_range.403] key 'MessageType' not found
[       OK ] RemoteStateHelperTest.TestSendEmptyMessage (2 ms)
[ RUN      ] RemoteStateHelperTest.TestSendValidMessageType
[       OK ] RemoteStateHelperTest.TestSendValidMessageType (8 ms)
[ RUN      ] RemoteStateHelperTest.TestSendMessageWithoutEndpointName
RemoteProvider: Invalid result: [json.exception.out_of_range.403] key 'EndpointName' not found
[       OK ] RemoteStateHelperTest.TestSendMessageWithoutEndpointName (3 ms)
[----------] 4 tests from RemoteStateHelperTest (17 ms total)
```

Reply:
Awesome!